bin/migrate-oats-data/applications/migrate_application.py

```diff
@@ -69,7 +69,7 @@ def process_application_etl(batch_size):
     update_application_incomplete_date(batch_size)
     process_application_parcels(batch_size)
     process_application_owners(batch_size)
-    process_application_applicant_on_submissions(batch_size)
+    # process_application_applicant_on_submissions(batch_size)
```

Comment:
Why comments and not deletions?

Reply:
currently working on debugging test, mekhti should have a separate MR addressing it soon